mapactionpy_arcmap/LayerProperties.py

```diff
@@ -1,23 +1,23 @@
-import json 
-import os
+import json
 from LayerProperty import LayerProperty
 
+
 class LayerProperties:
     def __init__(self, layerPropertiesJsonFile):
-        self.layerPropertiesJsonFile=layerPropertiesJsonFile
-        self.properties = set();
+        self.layerPropertiesJsonFile = layerPropertiesJsonFile
+        self.properties = set()
 
     def parse(self):
```

Comment:
I recommend calling `parse()` from within the constructor. I can't foresee a need to create a `LayerProperties` object (with the current constructor which requires a `layerPropertiesJsonFile` parameter), without needing to parse the json file. Since that would mean that the only use of the `parse()` method would be to call it from within the constructor, then it could also be renamed `_parse()`.

Also it would be appropriate to have a test that all of the layers named in the exist on the disk. As far as I can tell it would not be a valid state of to have a LayerProperties object which reference layerfiles which do not exist on disk. Hence this test should be called by the constructor. Hence it should be necessary to add a `lyr_files_dir` parameter to the constructor (or similar name). Some of this logic is currently in the `config_verify.py` file, though would be better if moved to the LayerProperties and MapCookbook.py files:
https://github.com/mapaction/mapactionpy_arcmap/pull/4/files#diff-01eac21ec2869a7f1f01805a2b1b0779